src/Refactoring-Core/RBRenameInstanceVariableRefactoring.class.st

```diff
@@ -92,6 +115,7 @@ RBRenameInstanceVariableRefactoring >> newName: anObject [
 
 { #category : 'preconditions' }
 RBRenameInstanceVariableRefactoring >> preconditions [
+
 	^(RBCondition isValidInstanceVariableName: newName for: class)
 		& (RBCondition definesInstanceVariable: variableName in: class)
 			& (RBCondition hierarchyOf: class definesVariable: newName) not
```

Comment:
Why not using the new condition (probably except the first one). 
To me isValidInstanceVariableName: should be called in the newName: accessor.

Reply:
Same as the next conflict